.gitignore

```diff
@@ -1,5 +1,6 @@
 /doc/
 /lib/*/ebin/
+/lib/*/ebin_reproducible/
```

Comment:
Given that we move it to the makefile, then I believe it is best for us to remove the directories later. Another idea is to write the ebins to `lib/*/tmp/ebin_reproducible`. Thoughts?

Reply:
still the task can be interrupted, and the files are going to be left behind.
I will look into the lib/*/tmp/ebin_reproducible option...

Reply:
Sorry, the comment above doesn't make sense. I wanted to say I don't like to have them potentially stay around in case of failures, which is why I would prefer to have them in `tmp`. :)

Reply:
> > @josevalim can you have a look at this and tell me what would be the cause of the error
> > https://travis-ci.org/elixir-lang/elixir/jobs/484534189#L755
> > could it be because we are not running make clear ?
> 
> I am not sure why. If I had to guess, it changed the time for the VM but not for the filesystem operations.

don't worry about it for now, was we are not getting that anymore.